src/keys/bip39.rs

```diff
@@ -60,7 +76,7 @@ impl<Ctx: ScriptContext> DerivableKey<Ctx> for Seed {
 impl<Ctx: ScriptContext> DerivableKey<Ctx> for MnemonicWithPassphrase {
     fn into_extended_key(self) -> Result<ExtendedKey<Ctx>, KeyError> {
         let (mnemonic, passphrase) = self;
-        let seed = Seed::new(&mnemonic, passphrase.as_deref().unwrap_or(""));
+        let seed: Seed = mnemonic.to_seed(passphrase.unwrap_or_else(|| "".to_string()));
```

Comment:
Wouldn't it be simpler this way?
```suggestion
        let seed: Seed = mnemonic.to_seed(passphrase.unwrap_or("".to_string()));
```

Reply:
Here `Seed` is just a type definition, not a structure. `rust-bip39` only requires `[u8; 64]` as the seed data. The key methods already had a `Seed` structure (from `tiny-bip39`). So I instead used a type alias. Thus we don't have any `Seed::new()` ctor. 

But agreed on the passphrase deref-dafault. That looks cleaner for `&str`.

Updated.

Reply:
Done..